src/test/java/net/sf/jabref/exporter/layout/format/DefaultTest.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.exporter.layout.format;
+
+import net.sf.jabref.exporter.layout.ParamLayoutFormatter;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DefaultTest {
+
+    @Test
+    public void testSimpleText() {
+        ParamLayoutFormatter a = new Default();
```

Comment:
I see the point. However, I believe that the file name naming makes it quite clear. (And the reason for keeping it is to quicker test new formatters...)